runtime/lua/builtin_util.lua

```diff
@@ -0,0 +1,90 @@
+
+local util = {}
+
+vim = vim or {}
+
+util.echom = function(message)
+  vim.api.nvim_command('echom "' .. tostring(message) .. '"')
+end
```

Comment:
This is a wrapper of common (n)vim functionality. Perhaps we should logically split it from pure-Lua functions that help us do something? If we had the liberty of changing the `vim` object I'd put it there. WDYT @ZyX-I ?

Reply:
See https://github.com/neovim/neovim/pull/8677